### PR TITLE
Updated UAT as per the latest changes

### PR DESCRIPTION
**Pull request by Praveengostu**

### CODE REVIEW

**Comment by sakshiag** [comment]
@Praveengostu : travis checks are failing ... Can you please check once and correct it..

**Comment by Praveengostu** [comment]
@sakshiag working on this

File: ibm/resource_ibm_network_gateway_vlan_attachment_test.go [hunk 1]
```diff
@@ -84,69 +39,24 @@ func TestAccIBMNetworkGatewayVlanAtachment_Basic(t *testing.T) {
...
 		PreCheck:  func() { testAccPreCheck(t) },
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: testAccCheckIBMNetworkGatewayVlanAttachment_import_update(gatewayName, hostname1, hostname1),
+				Config: testAccCheckIBMNetworkGatewayVlanAttachment_import_update(gatewayName, hostname1),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
-						"ibm_network_gateway_vlan_association.attachment", "bypass", "false"),
-					resource.TestCheckResourceAttr(
```
Comment (sakshiag) [inline ibm/resource_ibm_network_gateway_vlan_attachment_test.go]: Can you please add the check for number of members..

File: ibm/resource_ibm_network_gateway_vlan_attachment_test.go [hunk 2]
```diff
@@ -11,67 +11,22 @@ import (
...
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			resource.TestStep{
-				Config: testAccCheckIBMNetworkGatewayVlanAttachment_basic(gatewayName, hostname1, hostname1),
+				Config: testAccCheckIBMNetworkGatewayVlanAttachment_basic(gatewayName, hostname1),
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr(
 						"ibm_network_gateway_vlan_association.attachment", "bypass", "true"),
-					resource.TestCheckResourceAttr(
-						config, "members.#", "2"),
```
Comment (sakshiag) [inline ibm/resource_ibm_network_gateway_vlan_attachment_test.go]: Please add the check for number of members

Comment (Praveengostu) [reply ibm/resource_ibm_network_gateway_vlan_attachment_test.go]: @sakshiag Vlan attachment UAT is basically for validating that particular resource.. Gateway UAT has been taking care of validating Gateway resource

**Pull request merged**